source/settings.rst

```diff
@@ -906,8 +909,8 @@ depending on whether or not they are distributed with that repository.
 
       .. setting:: Vertical ruler position
 
-        Position for ruler.
-        .. todo Any effect?
+        Position for ruler in TextEditor controls.
+        (This should be moved to the TextEditor context menu.)
```

Comment:
Is there a special value that disables the ruler? If so can we document that here?